### PR TITLE
More on aff 4

### PR DESCRIPTION
**Pull request by justinwoo**
just PRing my hodge podge of packages. Could remove some that are more specific to my own needs.

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -1926,6 +2003,14 @@
     "repo": "https://github.com/natefaubion/purescript-variant.git",
     "version": "v4.0.0"
   },
+  "xstream": {
+    "dependencies": [
+      "aff-promise",
+      "js-timers"
+    ],
+    "repo": "git://github.com/justinwoo/purescript-xstream.git",
```
Comment (paf31) [inline packages.json]: `git` -> `https` here pls :)

File: packages.json [hunk 2]
```diff
@@ -1188,6 +1237,17 @@
...
   },
+  "node-telegram-bot-api": {
+    "dependencies": [
+      "aff",
+      "strings",
+      "foreign",
+      "aff-promise",
+      "simple-json"
+    ],
+    "repo": "git://github.com/justinwoo/purescript-node-telegram-bot-api.git",
```
Comment (paf31) [inline packages.json]: Another `git`

**Comment by oliver3** [comment]
Hi Justin,

Very possible that I just don't understand how things work, but to me it seems that `node-telegram-bot-api` v1.0.0 is still depending on purescript-aff ^3.0.0, so I am not sure how this package-set could work... Only way I see is that the 1.0.0 is actually not the same as the "bower version" 1.0.0 ?

**Comment by justinwoo** [comment]
Sorry, commented in the wrong spot at first. Yeah, seems strange to me that I was able to use the existing 1.0.0 in https://github.com/justinwoo/simple-rpc-telegram-bot/commit/c7199ca8d72344d6c5da8a30daa15dd03622c2e0  where the package set I used there very clearly points at 1.0.0, but it still builds fine. But I guess there's no problems since I use a very few of what is there?

**Comment by oliver3** [comment]
Ah thanks @justinwoo, also for your reply on the psc-package issue, and for releasing the telegram API

**Comment by LiamGoodacre** [comment]
Thanks!

**Pull request merged**